llama-index-networks/llama_index/networks/network/retriever.py

```diff
@@ -34,4 +47,14 @@ async def _aretrieve(self, query_bundle: QueryBundle) -> List[NodeWithScore]:
         contributor_results: List[List[NodeWithScore]] = await asyncio.gather(
             *async_tasks
         )
-        return reduce(lambda x, y: x + y, contributor_results)
+        nodes = reduce(lambda x, y: x + y, contributor_results)
+
+        for postprocessor in self._node_postprocessors:
+            nodes = postprocessor.postprocess_nodes(nodes, query_bundle=query_bundle)
```

Comment:
Note to self: add async node post processors lol

Reply:
Haha. Of course, how did I miss that. I'll add it quickly.

Reply:
oh nvm we don't have this yet?